snippets/perl/README.md

```diff
@@ -0,0 +1,28 @@
+### NetBackup API Code Samples for Perl
+
+This directory contains code samples to invoke NetBackup REST APIs using Perl.
+
+#### Disclaimer
+
+These scripts are only meant to be used as a reference. If you intend to use them in production, use it at your own risk.
+
+#### Pre-requisites:
+
+- NetBackup 8.1.1 or higher
+- Perl v5.18.2
+
+#### Executing the snippets in Perl
+
+Job Details:
+
+- Use the following command to obtain the job details from your NetBackup Master server:
+  - perl get_nb_jobs.pl -h `<masterServer>` -u `<username>` -p `<password>` [-d `<Domain name>`] [-t `<Domain type>`]
```

Comment:
Switches (-h/-u/-p etc) are not consistent with existing convention used in python/powershell snippets. I would suggest sticking to nbmaster/username/password etc. While you are at it, please review the switches from all existing files and make them consistent.

Reply:
done